_posts/2020-04-03-what-does-appsync-do.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title: What does AWS AppSync do ?
```

Comment:
Might be worth renaming this to

"Getting Started with AWS AppSync" or "Putting the Serverless in GraphQL with AppSync"